### PR TITLE
add import-export editor file (#60)

### LINKED ISSUE #60: Import end export feature for webapp
_We can use various input files in this import mode to setup an already created scene in Editor mode.
Autosave feature?
Export to main format or any other from the ones we choose._

An import-export feature similar to existing in the editor but should include zones.

### PR DESCRIPTION
**Pull request by ostatni5**
Added possibility to export state of the editor to a file and import state from a file.

obraz

Closes #60.

### CODE REVIEW

File: src/ThreeEditor/js/Menubar.File.js [hunk 1]
```diff
@@ -68,104 +67,31 @@ function MenubarFile(editor) {
...
-
-			output = JSON.stringify(output, null, '\t');
-			output = output.replace(/[\n\t]+([\d\.e\-\[\]]+)/g, '$1'); // eslint-disable-line
-
-		} catch (e) {
-
-			output = JSON.stringify(output);
-
-		}
+		const fileName = window.prompt('Input file name ', 'editor');
```
Comment (grzanka) [inline src/ThreeEditor/js/Menubar.File.js:91]: One may interpret this as the name of the input file.

File: src/ThreeEditor/js/Menubar.File.js [hunk 2]
```diff
@@ -56,7 +55,7 @@ function MenubarFile(editor) {
 
 	option = new UIRow();
 	option.setClass('option');
-	option.setTextContent(strings.getKey('menubar/file/import'));
+	option.setTextContent('Open');
```
Comment (grzanka) [inline src/ThreeEditor/js/Menubar.File.js:58]: If we have export, the this one should be called import
```suggestion
	option.setTextContent('Import');
```

**Review by grzanka — CHANGES_REQUESTED** [review]
Works well, I request only minor changes in naming

File: src/ThreeEditor/js/Menubar.File.js (hunk 2 above)
Comment (ostatni5) [reply src/ThreeEditor/js/Menubar.File.js:58]: But that action is erasing the current state in the editor. I think it is better to rename `Export` to `Save`.

Comment (grzanka) [reply src/ThreeEditor/js/Menubar.File.js:58]: Then what about renaming `Export` to `Clean and Save` ?

**Review by grzanka — DISMISSED** [review]
Looks ok

**Review by grzanka — APPROVED** [review]
looks ok

**Pull request merged**